lib/inet_ntop.c

```diff
@@ -38,7 +38,6 @@
 #include "curl_printf.h"
 
 #define IN6ADDRSZ       16
-#define INADDRSZ         4
```

Comment:
It's also a series, but I figured it's not critical, and hasn't been used since ~2010.

Reply:
Commented instead.